src/components/TrackMaintaining.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useCallback, Fragment } from 'react'
+
+import { useActionableState } from '../hooks/useActionableOnly'
+
+import { useView, setOptionsInUrl } from '../hooks/useUrlState'
+import { TrackAside } from './TrackAside'
+import { SwitchToggle } from './SwitchToggle'
+import { ViewSelectLink } from './ViewSelectLink'
+import { useRemoteConfig } from '../hooks/useRemoteConfig'
+import { TrackIcon } from './TrackIcon'
+import { TrackDescription } from './TrackDescription'
+import { ExerciseDetails } from './views/ExerciseDetails'
+import { LaunchList } from './views/LaunchList'
+import { ExerciseTree } from './views/ExerciseTree'
+
+export function TrackMaintaining({
+  trackId,
+}: {
```

Comment:
For consistency, you might want to extract this to `TrackMaintaingingProps`

Reply:
Will do!